test/apiv2/10-images.at

```diff
@@ -229,6 +234,8 @@ if ! grep -q '400 Bad Request' "${TMPD}/headers.txt"; then
     BUILD_TEST_ERROR="1"
 fi
 
+curl -XPOST --data-binary @<(cat )
```

Comment:
One more: please remove this, or fix it to be whatever you intended it to be. This is a bug.